synapse/handlers/room_member.py

```diff
@@ -286,7 +294,7 @@ def _update_membership(
 
             if not is_host_in_room:
                 inviter = yield self.get_inviter(target.to_string(), room_id)
-                if inviter and not self.hs.is_mine(inviter):
+                if inviter and not self.is_mine(inviter):
```

Comment:
is there a reason to change this?

Reply:
TBH I was mostly using it so that I could track down all the various places we were doing annoying `hs.get_*()` calls. Am happy to put them back